exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -141,7 +141,9 @@ static Span generateSpan(SpanData spanData, Endpoint localEndpoint) {
         });
     SpanData.Status status = spanData.getStatus();
     // for GRPC spans, include status code & description.
-    if (status != null && spanAttributes.get(SemanticAttributes.RPC_SERVICE) != null) {
+    if (status != null
```

Comment:
Don't believe status can actually be null, if it can we should fix it to return unset in such cases in a separate PR.